### PR TITLE
Custom security lab updates

### PR DESCRIPTION
**Pull request by ggbecker**
Update custom security content labs for ComplianceAsCode/content version 0.1.47.

### CODE REVIEW

File: 2019Labs/CustomSecurityContent/documentation/lab0_setup-devconf.adoc [hunk 1]
```diff
@@ -0,0 +1,114 @@
+:toc2:
+:linkattrs:
+:imagesdir: images
+
+= Lab Exercise 0: Setup Steps
+
+
+== 0.1 Accessing Your Dedicated Lab Environment Using Your Provided GUID
+
+. From your laptop, navigate to the https://www.opentlc.com/gg/gg.cgi?profile=generic_matyc[*Lab GUID Assignment page*^]. Firefox with plugins disabled recommended.
```
Comment (ggbecker) [inline 2019Labs/CustomSecurityContent/documentation/lab0_setup-devconf.adoc]: The only difference in this file from [lab0_setup-rhte.adoc](https://github.com/RedHatDemos/SecurityDemos/blob/master/2019Labs/CustomSecurityContent/documentation/lab0_setup-rhte.adoc) is that we changed the URL to use the profile `generic_matyc`

Comment (matejak) [reply 2019Labs/CustomSecurityContent/documentation/lab0_setup-devconf.adoc]: Plus, we would like to add instructions for people that use the local Fedora VM. I recall
- run playbooks
- in your head, swap rhel8 with fedora when following instructions

File: 2019Labs/CustomSecurityContent/documentation/lab1_introduction.adoc [hunk 2]
```diff
@@ -294,7 +294,7 @@ In the terminal, run the following commands:
 ----
 [... lab1_introduction]$ cd linux_os/guide/system/accounts/accounts-session/accounts_tmout
 [... accounts_tmout]$ ls
-ansible  bash  oval  rule.yml
+ansible  bash  oval  rule.yml  tests/
```
[the remark this answers is not in the document]
Comment (ggbecker) [reply 2019Labs/CustomSecurityContent/documentation/lab1_introduction.adoc]: done

File: 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc [hunk 3]
```diff
@@ -568,10 +568,10 @@ This way, you do not have to worry about possibly introducing those link:https:/
 . Finally, run the rule's test again--it may be that a typo was introduced, so the OVAL is not actually correct:
 +
 ....
-[... rule_accounts_tmout]$ cd /home/lab-user/labs/lab5_oval
+[... accounts_tmout]$ cd /home/lab-user/labs/lab5_oval
 [... lab5_oval]$ ./build_product rhel7
 ...
-[... lab5_oval]$ sudo tests/test_suite.py rule --container ssg_test_suite --datastream build/ssg-rhel7-ds.xml accounts_tmout
+[... lab5_oval]$ sudo python3 tests/test_suite.py rule --container ssg_test_suite --datastream build/ssg-rhel7-ds.xml accounts_tmout
```
[the remark this answers is not in the document]
Comment (ggbecker) [reply 2019Labs/CustomSecurityContent/documentation/lab5_oval.adoc]: done

**Comment by matejak** [comment]
Great job, merging!

**Pull request merged**